src/components/TheDestinationPicker.vue

```diff
@@ -1,21 +1,33 @@
 <script setup lang="ts">
 import AutoComplete from "primevue/autocomplete";
 import { TheParametersPicker } from ".";
-import { ref, watchEffect } from "vue";
 import {fetchGeocodingResults} from "../api/getElement"
+<<<<<<< Updated upstream
+import {testRouteCSM} from "../api/getCircularRoute"
+=======
+import {fetchCycleRouting} from "../api/getCycleRouting"
+>>>>>>> Stashed changes
+import { ref, watchEffect } from "vue";
+import console from "console";
 
 const from = ref("");
 const to = ref("");
+<<<<<<< Updated upstream
 const filteredCities = ref();
+=======
+>>>>>>> Stashed changes
+
+const filteredCities = ref();
+const test = async () => {
```

Comment:
Trochę nie pasuje mi że to się nazywa test. Trzeba dać gdzieś komentarz co to robi